docs/technical-documentation/docs_style_guide.md

```diff
@@ -0,0 +1,14 @@
+# Documentation Style Guide
+
+## Do's
+
+- Submit documentation formatted in [Markdown](https://en.wikipedia.org/wiki/Markdown) format.
+  -- Please add Markdown headings to the content sections.
+- Use a Github Pull Request to submit documentation.
+- Use the "bold/emphasis" style  in Markdown for UI elements that users will interact with. For example, a button label for a button that must be pressed should be made bold in Markdown.
+- Use the "italics" style in Markdown for UI elements that have a label or title if you need to reference them in the documentation. For example, a title of a screen or page that will visit should be made italic in Markdown.
+
+## Don'ts
+
+- Do not leave any "trailing spaces" at the end of lines of content.
+- Do not use "curly" quotes and apostrophes, use only "straight" quotes and apostrophes.
```

Comment:
Yeah, nice.  When you copy paste curly quotes into a terminal the command won't run and then you have to go fix them :hankey: